app/templates/suppliers/edit_company_registration_number.html

```diff
@@ -0,0 +1,89 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Company registration number – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {% with items = [
+    {
+      "link": "/",
+      "label": "Digital Marketplace"
+    },
+    {
+       "link": url_for('.dashboard'),
+       "label": "Your account"
+    },
+    {
+      "link": url_for('.supplier_details'),
+      "label": "Company details"
+    },
+  ] %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="single-question-page">
+  <div class="grid-row">
+    <div class="column-one-whole">
+      {% if form.errors %}
+        {% for field in form.errors %}
```

Comment:
This feels weird because it looks like you could end up with multiple validation boxes at the top of the page. It would be nice if we don't have a for loop around this as we anticipate only ever displaying one (and it would be weirdly broken to show more).

I think it should be a list comprehension inside the `{% with ... %}` block.

Reply:
You can't do list comprehensions in Jinja, but I'll see if I can find a nicer way.

Reply:
Oh wow, yeah, totally forgot that. Maybe can build the structure Python side. 🤷‍♂️